### ISSUE TITLE
Add state/county level `meta` headers

### ISSUE THREAD

**Issue by asciimike** [covid-projections/covid-projections#299]
Current `meta` tags on every page:

```html
<meta name="title" content="This model predicts the last day each state can act before the point of no return" />
<meta name="description" content="The only thing that matters right now is the speed of your response" />
```

We want to provide per state/county meta information when Googlebot scrapes the page. It should also populate previews and share cards with more accurate information.

Titles:
 - "Shelter in place": `Maintain shelter in place in {{stateOrCounty}}.`
 - All others: `You must act now in {{stateOrCounty}}.`

Note: I expect that we have counties suffixed with "county", so I'd expect copy for a non-shelter-in-place county to read: "You must act now in Vigo County."

Descriptions:
 - "Shelter in place": `Avoiding hospital overload depends heavily on your cooperation.`
 - All others: `To prevent hospital overload, our projections indicate shelter in place must be implemented soon.`

Consider something like https://github.com/nfl/react-helmet

**Comment by mikelehen** [comment]
This was fixed via #306.

**Comment by szhu** [comment]
Related: #411
